### ISSUE TITLE
When looking in XDG_CONFIG_HOME YAML config file shouldn't have dot prefix

### ISSUE THREAD

**Issue by cpjolicoeur** [achannarasappa/ticker#40]
When storing configuration files in XDG_CONFIG_HOME, it's not typical to also have the configuration file be a hidden file.

e.g. 

    ~/.ticker.yaml
    ~/.config/ticker/ticker.yaml

This issue is just a placeholder to make the config file loader a bit more robust in how it search/loads the config file to allow for a bit more leeway.  The README does note the file being called `.ticker.yaml` in both cases, but it would also be nice (perhaps) to allow for both styles.

**Comment by boreq** [comment]
I agree this is highly unusual and confusing. It took me a long while to find the config file. Additionally the file should be placed in a separate directory `ticker` within `$XDG_CONFIG_HOME`. If the standard isn't followed then maybe it is better not to try to follow it at all.
